tasks/build/browserify.js

```diff
@@ -1,66 +1,31 @@
 'use strict';
 
 var path = require('path');
-var browserify = require('browserify');
-var derequire = require('derequire');
-
-var bannerTemplate =
-  '/*! p5.js v<%= pkg.version %> <%= grunt.template.today("mmmm dd, yyyy") %> */';
+const Bundler = require('parcel-bundler');
 
 module.exports = function(grunt) {
-  var srcFilePath = require.resolve('../../src/app.js');
-
-  grunt.registerTask(
-    'browserify',
-    'Compile the p5.js source with Browserify',
-    function(param) {
-      var isMin = param === 'min';
-      var filename = isMin ? 'p5.pre-min.js' : 'p5.js';
-
-      // This file will not exist until it has been built
-      var libFilePath = path.resolve('lib/' + filename);
-
-      // Reading and writing files is asynchronous
-      var done = this.async();
-
-      // Render the banner for the top of the file
-      var banner = grunt.template.process(bannerTemplate);
-
-      // Invoke Browserify programatically to bundle the code
-      var browseified = browserify(srcFilePath, {
-        standalone: 'p5'
-      });
-
-      if (isMin) {
-        browseified = browseified.exclude('../../docs/reference/data.json');
-      }
```

Comment:
@Spongman that's now handled by `process.env.IS_MINIFIED` since if that is `false`, `data.json` is never `require`'d in the bundle (see https://github.com/processing/p5.js/pull/3431/files#diff-52d252b9f74ec945f5b64c3f53be90f1R15).